monai/networks/layers/convutils.py

```diff
@@ -78,13 +78,17 @@ def calculate_out_shape(
     return out_shape if len(out_shape) > 1 else out_shape[0]
 
 
-def gaussian_1d(sigma: Union[float, torch.Tensor], truncated: float = 4.0) -> np.ndarray:
+def gaussian_1d(sigma: Union[float, torch.Tensor], truncated: float = 4.0, approx: str = "simple"):
     """
-    one dimensional gaussian kernel.
+    one dimensional Gaussian kernel.
 
     Args:
         sigma: std of the kernel
         truncated: tail length
+        approx: Discrete Gaussian kernel type, available options are "simple" and "refined".
```

Comment:
I would rather refer to these as "erf" (instead of "simple") and "scale-space" (instead of refined)

Reply:
It would also be good to have an even simpler "sampled approach", e.g.
```python
    if approx.lower() == "sampled":
        x = torch.arange(-tail, tail + 1).float()
        t = 1 / (torch.tensor(2.0*np.pi).sqrt() * sigma)
        out = t* torch.exp((-x.pow(2.0) / float(2 * sigma ** 2)))
        return out.clamp(min=0)
```